pkgs/development/python-modules/passlib/default.nix

```diff
@@ -15,9 +16,17 @@ buildPythonPackage rec {
     sha256 = "defd50f72b65c5402ab2c573830a6978e5f202ad0d984793c8dde2c4152ebe04";
   };
 
-  checkInputs = [ nose ];
-  propagatedBuildInputs = [ bcrypt argon2_cffi ];
-  propagatedNativeBuildInputs = [ argon2_cffi ];
```

Comment:
This is now lost which I think was really important.

Reply:
Those are extra-requires.

Reply:
Yes but extra-requires currently is not easy to use with propagatedNativeBuildInputs. I am not sure how we want to make sure people actually add both.

Reply:
Why is propagatedNativeBuildInputs needed?

Reply:
For cross-compilation https://github.com/NixOS/nixpkgs/commit/1cb011217d68bb068a9b9a03f6330d5778c86738

Reply:
It's not perfect, but not depending on argon2_cffi at all should also fix cross-compilation.

Reply:
cc @legendofmiracles